pulsar-common/src/main/java/org/apache/pulsar/common/util/RateLimiter.java

```diff
@@ -92,6 +92,11 @@ public RateLimiter(final ScheduledExecutorService service, final long permits, f
         this(service, permits, rateTime, timeUnit, permitUpdater, false);
     }
 
+    public RateLimiter(final ScheduledExecutorService service, final long permits, final long rateTime,
```

Comment:
RateLimiter already contains a lot of different constructors. The alternative is to to use the builder generated with the Lombok Builder annotation. Example of using this is in https://github.com/apache/pulsar/blob/ded806fd52f6e2f182fa02052cbd82c2a6755098/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PrecisPublishLimiter.java#L101-L117

Reply:
ok,I will fix this.